picard/formats/id3.py

```diff
@@ -269,142 +291,209 @@ def build_TXXX(self, encoding, desc, values):
 
     def _load(self, filename):
         log.debug("Loading file %r", filename)
+        tags, config_params = self._init_load(filename)
+
+        self._upgrade_23_frames(tags)
+
+        metadata = Metadata()
+        for frame in tags.values():
+            self._process_frame(frame, metadata, config_params)
+
+        if 'date' in metadata:
+            self._sanitize_date(metadata)
+
+        self._info(metadata, config_params['file'])
+        return metadata
+
+    def _process_frame(self, frame, metadata, config_params):
+        """Process an ID3 frame and add its data to the metadata."""
+        frameid = frame.FrameID
+
+        # Get appropriate processor from dictionary
+        processor = self._frame_processors.get(frameid)
+        if processor:
+            processor(frame, metadata, config_params)
+
+    def _init_load(self, filename):
```

Comment:
Where is this called?

Reply:
https://github.com/metabrainz/picard/pull/2590/files#diff-1bde7f3f6de30b8d739c17c9e528a07ee6c5f511ac73846855fdfcb36dd39cb2R294